website/docs/docs/use-dbt-semantic-layer/setup-sl.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "Set up the dbt Semantic Layer"
+id: setup-sl
+description: "Seamlessly set up the dbt Semantic Layer in dbt Cloud using intuitive nagivation."
```

Comment:
sugg: consider removing or using diff words than "seamlessly" and "intuitive". in techdocs, the motivation is similar to using "easier" instead of "easy". this is because what if the person doesn't find it easy, reading the word "easy" could just annoy or add to their frustration (ie it might not seem "seamless" or "intuitive" to the person and we should be mindful of that). 

```suggestion
description: "Seamlessly set up the dbt Semantic Layer in dbt Cloud using intuitive navigation."
```